### ISSUE TITLE
Add documentation in additional languages

### ISSUE THREAD

**Issue by reubenae** [intuit/user-data-for-fraud-prevention#113]
**Is your feature request related to a problem? Please describe.**
This project could be more inclusive if the readme was available in multiple languages.

**Describe the solution you'd like**
Create copies of README.md in the project root, named appropriately e.g. README_SPANISH.md and linked to from the main readme. Something is better than nothing, so provided a single section of the readme is fully translated, that is sufficient to create a new file.

**Describe alternatives you've considered**
I'm open to other ideas on how we can make this project more inclusive, and any improvements to the process described above.

**Additional context**

**Comment by 12-malak** [comment]
@reubenae Hi, I would like to work on this. Please assign me under Open Source Day 2021.

**Comment by reubenae** [comment]
Hi @12-malak no need for assigning, there are plenty of translations to go around 🙂 we don't assign tickets for this repo

You'll get credit for the PR on its own.

**Comment by KerinPithawala** [comment]
Can I add setup a workflow in this repository to add additional languages?

**Comment by reubenae** [comment]
@KerinPithawala if you think it will help with this issue, then please go for it and let me or @skodamarthi know!

**Comment by KerinPithawala** [comment]
Yes @reubenae I can set up an workflow to translate and create this readme.md file to Chinese traditional, Arabic, Hindi and French.

**Comment by reubenae** [comment]
@KerinPithawala sounds good to me. Let us know when you have something ready for us to review!

**Comment by skodamarthi** [comment]
@reubenae Looks like @KerinPithawala is going to use https://github.com/dephraiim/translate-readme for automatic translation using https://cloud.google.com/translate/docs/languages. Is automatic translation what we want?

**Comment by reubenae** [comment]
Oh... no. We want documentation that is approved by someone who can write in the language fluently. I'm happy for google translate to be used as a start but unless you can write fluently in that language and will be able to correct any mistakes, I think we wouldn't accept the contribution. What do you think @skodamarthi?

**Comment by skodamarthi** [comment]
Yes... I agree too! @KerinPithawala Apologies for the confusion! If you can validate the google translation and correct it to make it meaningful for readers in the language you are fluent in, we can accept that contribution for that language. At the moment, we are not comfortable adding automatic translations. Please let us know if you have any questions.

**Comment by KerinPithawala** [comment]
Makes sense @skodamarthi @reubenae . I don't know these languages well. It would be better if we get someone who is aware of them.

**Comment by skodamarthi** [comment]
No problem @KerinPithawala! Thank you for looking into this issue. This discussion would clarify the expectation for other contributors too. If you would like, you can still contribute to add readme file in the language you are fluent in.

**Comment by KerinPithawala** [comment]
Sure. Thanks!

**Comment by someOne404** [comment]
I can work on adding simplified Chinese and traditional Chinese

**Comment by manueldose** [comment]
Hi @reubenae do you still need help with the Spanish translation? I speak Spanish natively. I'm starting in the openSource world and I 'd like to help. Do you need the modification of the README.md file?

**Comment by reubenae** [comment]
@manueldose yes, absolutely! check out this PR https://github.com/intuit/user-data-for-fraud-prevention/pull/128 for a pattern you can follow (add it to the translation section on the English readme, create a new file for the spanish version). 

We'll add hacktoberfest-accepted labels to PR's once they're merged, so it'll all count for hacktoberfest

**Comment by manueldose** [comment]
Ok cool i'll get on with It!

**Comment by salilbc** [comment]
i can work on adding the Readme in Hindi, do you need the ReadMe in Hindi?
